package.json.orig

```diff
@@ -0,0 +1,1731 @@
+{
```

Comment:
I don't think you should need this file or any of the yarn changes. Could you please try removing these changes and ensure the branch still works?

Reply:
done